README.md

```diff
@@ -1,25 +1,87 @@
-# bitcoind.js
+bitcoind.js
+=======
+[![Build Status](https://img.shields.io/travis/bitpay/bitcoind.js.svg?branch=master&style=flat-square)](https://travis-ci.org/bitpay/bitcoind.js)
+[![Coverage Status](https://img.shields.io/coveralls/bitpay/bitcoind.js.svg?style=flat-square)](https://coveralls.io/r/bitpay/bitcoind.js)
 
 A Node.js module that adds a native interface to Bitcoin Core for querying information about the Bitcoin blockchain. Bindings are linked to Bitcore Core compiled as a shared library.
 
+## Install
+
+```bash
+git clone git@github.com:bitpay/bitcoind.js.git
+cd bitcoind.js
+npm install
+```
+
 ## Example Usage
 
-``` js
-var bitcoind = require('bitcoind.js')({
+```js
+
+var BitcoinNode = require('bitcoind.js');
+
+var configuration = {
   directory: '~/.bitcoin',
   testnet: true
+};
+
+var node = new BitcoinNode(configuration);
```

Comment:
We'll need to revisit this, and make sure the configuration options are passed through correctly. Spawned issue: https://github.com/bitpay/bitcoind.js/issues/40